mod/research/set/self/research_page/navigation.rb

```diff
@@ -6,14 +6,18 @@ def next_button type
 
   # @param type [:metric, :company]
   def previous_button type
-    navigate_button type, icon_tag(:chevron_left), previous_item(type)
+    navigate_button type, icon_tag(:chevron_left), previous_item(type), "ml-auto"
   end
 
-  def navigate_button type, text, item
-    return unless item
-
-    link_to text, path: research_url(type => item),
-                  class: "btn btn-outline-secondary btn-xs-icon mx-2"
+  def navigate_button type, text, item, extra_class=nil
+    opts = { class: "btn btn-outline-secondary btn-xs-icon" }
+    add_class opts, extra_class if extra_class
+    if item
+      opts[:path] = research_url(type => item)
+    else
+      add_class opts, "disabled"
+    end
+    link_to text, opts
```

Comment:
I take it we decided to make the navigate button always there (but disabled when not applicable?)

Reply:
that was the quickest fix to avoid that the buttons move around when one of them is missing